language-support/hs/bindings/src/DA/Ledger/Convert.hs

```diff
@@ -161,6 +165,23 @@ optional = \case
     Left _ -> Nothing
     Right a -> Just a
 
+raiseGetLedgerConfigurationResponse :: LL.GetLedgerConfigurationResponse -> Perhaps LedgerConfiguration
+raiseGetLedgerConfigurationResponse =
+    (perhaps "ledgerConfiguration" >=> raiseLedgerConfiguration)
```

Comment:
Mixing `.` and `>=>` looks rather confusing since they are both basically function composition but in the opposite direction. I would recommend that you either use `.` and `<=<` or you just go for do notation.